content/en/docs/concepts/services-networking/service.md

```diff
@@ -84,8 +84,7 @@ number that pods expose in the next version of your backend software, without
 breaking clients.
 
 `TCP` is the default protocol for services, and you can also use any other
-[supported protocol](#protocol-support). At the moment, you can only set a
-single `port` and `protocol` for a Service.
+[supported protocol](#protocol-support). As many `Services` need to expose more than one port, Kubernetes supports multiple port definitions on a `Service` object. Each port definition can have the same or a different `protocol`.
```

Comment:
To follow the style guide, “Services” should be initial caps but not in backticks. “protocol” in backticks seems fine to me: it's talking about how you configure the Service object, not the `service` section of some YAML.

Reply:
Consider wrapping this line to match the rest of the file.

Reply:
@sftim Thanks for your review. Fixed formatting issues as per your suggestions. Please have a look and let me know if you think more changes required